docs/install-config/harbor-compatibility-list.md

```diff
@@ -50,6 +50,7 @@ This document provides compatibility information for all Harbor components.
 | [Sysdig Secure](https://github.com/sysdiglabs/harbor-scanner-sysdig-secure) |![Sysdig](../../img/scanners/sysdig.png) | Sysdig  | ![Y](../../img/replication-adapters/right.png) |  | v2.1.0 |
 | [TensorSecurity](https://github.com/tensorsecurity/harbor-scanner) |![TensorSecurity](../../img/scanners/tensorsecurity.png) | TensorSecurity | ![Y](../../img/replication-adapters/right.png) |  | v2.2.0 |
 | [ArksecScanner](https://github.com/arksec-cn)    |![Arksec](../../img/scanners/arksec.png)| Arksec    |![Y](../../img/replication-adapters/right.png)| | v2.4.0 |
+| [Cyberwatch](https://github.com/Cyberwatch)    |![Cyberwatch](../../img/scanners/cyberwatch.png)| Cyberwatch    |![N](../../img/replication-adapters/no.png)|![Y](../../img/replication-adapters/right.png)| v2.2.0 |
```

Comment:
I think that should be 2.8.0

Reply:
also can you make the providers column(3rd) as link to the official docs from your site! https://docs.cyberwatch.fr/en/use_assets/harbor_scan.html

Reply:
@OrlinVasilev just updated the PR with required changes :)

Please note that I used this link https://cyberwatch.fr/integrate-with-harbor-scans which redirects 302 to https://docs.cyberwatch.fr/en/use_assets/harbor_scan.html so that we could update the documentation on our side without breaking anything on Harbor docs. If we change our doc, we will just update the 302 redirection target.